project/recursive_sorting.py

```diff
@@ -48,48 +48,31 @@ def quick_sort(arr, low, high):
     if low >= high:
         return arr
     else:
-        pivot = (low + high) // 2
+        pivot_val = arr[(low + high) // 2]
```

Comment:
nice! this is all a lot more concise! and nice going using left and right flags.